extensions/CommentingControllerTwitterUsernameExtension.php

```diff
@@ -0,0 +1,11 @@
+<?php 
+
+class CommentingControllerTwitterUsernameExtension extends DataExtension {
```

Comment:
As per subsequent note below I think the Twitter username case could fall into the same category as having a Gravatar.  My preference is having both available as optional extensions within the comments module, in that the template rendering a comment needs to know if the extensions are enabled or not.  This also helps regarding ripping out some of the old config.